book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,173 @@
+It covers such things a regulatory compliance that is not yet touched on here.
+It does occasionally appear to conflate open-source solutions with self-hosted ones which need not necessarily be the case.
+Some companies will let you host proprietary apps on premises and conversely you can pay for 3rd party hosting and administration of open source applications.
+This is important as if you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution you can still pay a 3rd party to do this for you. 
+In This case you get the benefits of professional support and the reassurance of an open solution.
+When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenence, for example by donating to the project or if it makes sense given the makeup of your organisation and that of the project contributing developer time.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers as the hosting and support are no longer vertically integrated parts of the software as a service (SaaS) experience for you.
+
+There are many solutions to choose from: [How to pick an electronic laboratory notebook](https://doi.org/10.1038/d41586-018-05895-3) by {cite:ps}`Kwok2018eln` or [2019 Review of the Best Electronic Laboratory Notebooks](https://app.scientist.com/blog/2019/04/05/2019-review-of-the-best-electronic-laboratory-notebooks) may help you to find your solution!
+This [guide on choosing an ELN from Simon Bungers of Labfolder](https://labfolder.com/electronic-lab-notebook-eln-research-guide/) is also worth a read.
+The Harvard Longwood Medical Area Research Data Management Working Group Constructed an [Electronic Lab Notebook Comparison Matrix](https://doi.org/10.5281/zenodo.4723753) in 2021, for convenience [a nicely formatted google sheet version](https://docs.google.com/spreadsheets/d/1ar8fgwagOh30E31EAPL-Gorwn_g6XNf81g3VDQnQ_I8/edit#gid=0), is also available.
+
+Note that ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities such as bioimaging facilities who can do things like automate a substancial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+ELNs offer many advantages over paper notebooks but they come with a considerable amount of additional complexity and a number of additional risks that it is important to have a plan to mitigate when proceeding with their adoption.
```

Comment:
I would delete this paragraph or move it to the section below since it is not really about choosing a certain ELN solution.